cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/deploy/InMemoryConfigurator.java

```diff
@@ -104,6 +108,7 @@ public InMemoryConfigurator(CConfiguration cConf, PluginFinder pluginFinder, Imp
     this.applicationName = deploymentInfo.getApplicationName();
     this.applicationVersion = deploymentInfo.getApplicationVersion();
     this.configString = deploymentInfo.getConfigString() == null ? "" : deploymentInfo.getConfigString();
+    this.changeSummary = deploymentInfo.getChangeSummary() == null ? "" : deploymentInfo.getChangeSummary();
```

Comment:
Change summary can be optional - A user is not prompted for change summary when the pipeline is created for the first time.